### PR TITLE
Fix stereo recording

### PR DESCRIPTION
**Pull request by daniilpastukhov**
- Fixed the errors that didn't allow user to record depth/disparity using `oak.visualize(..., record='...'))`
- Also fixed the problem with recording disparity data, and saving depth data (they were saved as `file_name.None`, now it's `file_name.dat`

### CODE REVIEW

**Comment by Erol444** [comment]
I would suggest targeting thi to `develop` branch after we merge https://github.com/luxonis/depthai/pull/827

File: depthai_sdk/examples/StereoComponent/stereo_record.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        wls_lambda=8000,
+        wls_sigma=1.5
+    )
+
+    # Record RGB and disparity to records folder
+    # Record doesn't work with visualize so the config is ignored
+    oak.record([color.out.main, stereo.out.disparity], 'records')
+
+    # Record depth only
+    # oak.visualize(stereo.out.depth, record='depth.avi')
```
Comment (Erol444) [inline depthai_sdk/examples/StereoComponent/stereo_record.py:30]: Do common video players (eg VLC) work with int16 pixels?

File: depthai_sdk/examples/StereoComponent/stereo_record.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import cv2
+import depthai
+
+from depthai_sdk import OakCamera
+from depthai_sdk.visualize.configs import StereoColor
+
+
+def callback(msgs, visualizer):
+    for name, packet in msgs.items():
+        cv2.imshow(packet.name, packet.frame)
```
Comment (Erol444) [inline depthai_sdk/examples/StereoComponent/stereo_record.py:10]: Is this used anywhere?

File: depthai_sdk/examples/StereoComponent/stereo_record.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+with OakCamera(usbSpeed=depthai.UsbSpeed.HIGH) as oak:
```
Comment (Erol444) [inline depthai_sdk/examples/StereoComponent/stereo_record.py:13]: Lets not force USB2 in an example

File: depthai_sdk/examples/StereoComponent/stereo_record.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import cv2
```
Comment (Erol444) [inline depthai_sdk/examples/StereoComponent/stereo_record.py:1]: do you think this script should go to `recording` folder instead?

**Review by Erol444 — APPROVED** [review]

File: depthai_sdk/examples/StereoComponent/stereo_record.py (hunk 1 above)
Comment (daniilpastukhov) [reply depthai_sdk/examples/StereoComponent/stereo_record.py:30]: FFV1 codec supports 16-bit grayscale frames, I didn't test if it works well with video players (but VLC should support it). Now depth data is saved in binary format, I tried to use ffv1 codec with the current implementation but it didn't work. I'll spend more time on this later.

File: depthai_sdk/examples/StereoComponent/stereo_record.py (hunk 2 above)
Comment (daniilpastukhov) [reply depthai_sdk/examples/StereoComponent/stereo_record.py:10]: Removed

File: depthai_sdk/examples/StereoComponent/stereo_record.py (hunk 3 above)
Comment (daniilpastukhov) [reply depthai_sdk/examples/StereoComponent/stereo_record.py:13]: Yeah, I removed it

File: depthai_sdk/examples/StereoComponent/stereo_record.py (hunk 4 above)
Comment (daniilpastukhov) [reply depthai_sdk/examples/StereoComponent/stereo_record.py:1]: Moved it to another folder

**Comment by daniilpastukhov** [comment]
@Erol444 I'm merging this PR and creating a new one that includes more things, all the changes will be there

**Pull request merged**